solidity/test/bridge/Bridge.test.ts

```diff
@@ -54,6 +54,10 @@ const fixture = async () => {
   await bank.updateBridge(bridge.address)
   await bridge.connect(deployer).transferOwnership(governance.address)
 
+  // Set the redemption dust threshold to 0.001 BTC, i.e. 10x smaller than
+  // the initial value in the Bridge in order to save test Bitcoins.
+  await bridge.setRedemptionDustThreshold(100000)
```

Comment:
I am not following. Why can't we operate on existing numbers, BTC transactions we used for tests already happened.

Reply:
There are two reasons. First, we have a very limited amount of testnet BTC and we cannot easily recover everything so it is far easier to operate on smaller amounts. Second, some BTC transactions are used in multiple scenarios where different requests are made against. Having a smaller dust threshold makes reusing easier because the possible amount range is wider. This is especially useful in next PRs https://github.com/keep-network/tbtc-v2/pull/153 and https://github.com/keep-network/tbtc-v2/pull/155